### DISCUSSION TITLE
Auch hier Probleme bei der Verbindung zwischen DTU und HM1500

### DISCUSSION THREAD

**Discussion by Messiah246** [tbnobody/OpenDTU#1918] [Q&A] [answered]
Hallo zusammen,

ich habe diverse Threads schon rauf und runter gelesen und auch schon einiges probiert.
Im allgemeinen geht man davon aus, dass ein neues Funkmodul hilft und bei den Meisten war das wohl auch die Lösung.

Jetzt zu meinem Fall:

Vorhanden seit ca. einem Jahr Ahoi DTU auf ESP8266 mit Funkmodul nrf24l01+ - Funktioniert tadellos

Dann vor zwei Wochen neu bestellt:

-ESP32 Wroom Board und ein Funkmodul. Anschlüsse direkt über kurze Adern verlötet.
-ESP geflashed, pin mapping geladen und richtiges Profil gespeichert.
-Funkmodul wird erkannt, aber keine Verbindung zum Wechselrichter

Ok dachte ich, hab ich wohl was falsch gemacht. Alles kontrolliert, nochmal geflashed ... nada keine Funktion

Nach Recherche zu dem Schluss gekommen, dass es wohl am Funkmodul liegen wird.
Also 4 Stück bestellt, Funkmodul gewechselt - wieder keine Verbindung.

Funktionierende Ahoi DTU aus meinem Schaltkasten ausgebaut Funkmodul abgelötet an den ESP32 angelötet:
Ihr kommt sicher selber drauf - keine Funktion...

Das Funkmodul was ich gerade beim ESP32 abgelötet habe, habe ich anschließend an den ESP8266 gelötet und auch mit diesem Funkmodul funktioniert es mit Ahoi DTU.

Jetzt sagt mir mein gesunder Menschenvertsand an den Funkmodulen liegt es wohl nicht.

Verdrahtet habe ich nach folgendem Bild:

![8264242978c38d1dcbece30e048f1e16f9bd1554_2_1024x835](https://github.com/tbnobody/OpenDTU/assets/167350640/49b8efef-7301-4c71-8fdd-ca4c52ee0f4c)

Fehlt mir da irgendwo etwas, oder habe ich gar den falschen Belegungsplan benutzt?

- 47µF Kondensator ist testweise verlötet
- USB Netzteil mit 3A probiert
- Entfernung ESP32 zum Wechselrichter testweise unter 2m
- der ESP8266 mit Ahoi funktioniert in etwa 20m Entfernung durch mehrere Wände noch
- ESP32 Platine habe ich nach den Tests mit Ahoi ebenfalls noch eine bestellt. Auch mit der zweiten das Gleiche.

Hat noch jemand eine Idee was ich falsch gemacht haben könnte? Vielleicht ist im PIN mapping ein Fehler?

![info](https://github.com/tbnobody/OpenDTU/assets/167350640/f1c3bf64-3ce0-46a9-b933-3b25e8aa58e5)
![info2](https://github.com/tbnobody/OpenDTU/assets/167350640/80e467a1-b5ee-453f-a9ec-ce86aa1e153f)

Vielen Dank an Alle im voraus.

**Comment by tbnobody** [comment, marked as answer]
Du hast den IRQ pin nicht angeschlossen. 
Siehe: https://www.opendtu.solar/hardware/esp32nodemcu_38pin/#symbolic-view

Dadurch wird zwar der NRF chip erkannt, jedoch werden keine ankommenden Pakete signalisiert.

**Reply by Messiah246** [reply]
Vielen Dank für die schnelle Antwort.
Ich ging nach dem Verdrahtungsplan von oben davon aus, dass ich ihn nicht benötigen würde.

Jetzt hab ich aber das Problem, das meine esp32 anders beschriftet sind als in deinem Schaltplan. An welchen Kontakt müsste der IRQ Anschluss in dem Bild oben? D22?

Und muss ich dann etwas im mapping ändern?

**Reply by MetaChuh** [reply]
nachdem auf deinem board anscheinend d/gpio 16 nicht herausgeführt ist:
ja zb. d22 nehmen und dann die pin_mapping.json anpassen (irq von 16 auf 22 ändern)

**Reply by Messiah246** [reply]
Manchmal sieht man den Wald vor Bäumen nicht.
Danke für eure Hilfe. 

Ich hab mir jetzt einfach die Pin-Belegung vom Chip selber gesucht und hab mit dem Multimeter über die Durchgangsmessung den entsprechenden Kontaktpunkt an der Platine ausgemessen.

**Reply by MetaChuh** [reply]
@Messiah246 

bitte markier die für dich relevanteste antwort, egal welche, auch wenn es deine eigene ist 😉
das hilft anderen usern bei der suche, da in der regel zuerst die matching answered discussions gelesen werden.

thx & greetings
